Source/CsDebugScript.Engine/Engine/Debuggers/DbgEngDllHelpers/DebugCallbacks.cs

```diff
@@ -0,0 +1,109 @@
+﻿using DbgEngManaged;
+using System;
+
+namespace CsDebugScript.Engine.Debuggers.DbgEngDllHelpers
+{
+    /// <summary>
+    /// Debug callbacks called during WaitForEvent callback.
+    /// This class in future can be extended to support callbacks provided
+    /// on certain actions (e.g. breakpoint hit, thread create, module load etc.)
+    /// </summary>
+    class DebugCallbacks : IDebugEventCallbacks
+    {
+        /// <summary>
+        /// IDebugClient.
+        /// </summary>
+        private IDebugClient client;
+
+        /// <summary>
+        /// Constructor.
```

Comment:
Done.

---
In reply to: [69983250](https://github.com/southpolenator/WinDbgCs/pull/8#discussion_r69983250) [](ancestors = 69983250)